STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,69 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Technical Oversight Committee to exclusively focus on the technical aspects of the project. 
+
+## Steering Committee Charter
+
+1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio.
+2. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
+3. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository.
+4. Control and delegate access to and establish processes regarding other project resources and assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
+6. Resolve any dispute from the Technical Oversight Committee.
+
+## Steering Committee Members
+
+Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google & IBM. These are:
+
+#### Google
+
+* [April Nassi](https://github.com/thisisnotapril) (Google)
+* [Dan Ciruli](https://github.com/oaktowner) (Google)
+* Jennifer Lin (Google)
+* [Louis Ryan](https://github.com/louiscryan) (Google)
+* [Martin Taillefer](https://github.com/geeknoid) (Google)
+* [Sven Mawson](https://github.com/smawson) (Google)
+
+#### IBM
+* [Dan Berg](https://github.com/dcberg) (IBM)
+* [Lin Sun](https://github.com/linsun) (IBM)
+* Nilesh Patel (IBM)
+
+## Meetings
+
+The Istio Steering Committee meets weekly; currently on Tuesday mornings. 
+
+Given the nature of the discussions in Steering, meetings are not currently open to the public. Questions around governance are listed as [issues in the community repo](https://github.com/istio/community/labels/steering-governance), and we invite your feedback there. 
+
+If you wish to present to the Steering committee or have a topic you'd like Steering to review, please file an issue here or reach out via [istio-steering@](https://groups.google.com/forum/#!forum/istio-steering). 
```

Comment:
nit: can we spell out istio-steering@googlegroups.com here rather than just saying istio-steering@? Linking to the group doesn't add a ton of value for anyone not in the group since you just get an access denied page that doesn't even have the group's email address.

<img width="750" alt="screen shot 2018-10-03 at 2 31 52 pm" src="https://user-images.githubusercontent.com/1923135/46440760-3f01ea00-c719-11e8-9e60-13d2bc047007.png">

Reply:
Or make the link `mailto:istio-steering@googlegroups.com`

Reply:
Done